draft-irtf-cfrg-randomness-improvements.md

```diff
@@ -198,7 +204,11 @@ or use an independent (and completely reliable) entropy source, e.g., if Sig is
 in an HSM with its own internal trusted entropy source for signature generation.
 
 In systems where signature computations are expensive, G'(n) may be precomputed and pooled.
-This is possible since the construction depends solely upon the CSPRNG output and private key.
+This is possible since the construction depends solely upon the CSPRNG output and private key. 
+Experimental results indicate that caching the signature output is critical for performance. 
+The signature computation cost is amortized in about 1000 calls, afterwards each call of G’(n) 
```

Comment:
This number probably doesn't mean much without a description of the implementation and experimental platform. Can we move this bit to a separate PR where we can address that in full?

Reply:
I agree that mentioning "about 1000" without any information about the experiments may look quite lame. My opinion here was that a particular number is not necessary - only the idea of "cache the signature, or it will be really slower" should be reflected.

But you are right that it doesn't look good. I'll remove this one from this PR - let's discuss, how to add details about the experiments separately (taking into account adding an explicit reference to your paper with experiments).